src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionContainerBase.cs

```diff
@@ -140,7 +142,13 @@ private set
             _serversPing = new CustomizedPingTimer(Logger, Constants.CustomizedPingTimer.Servers, WriteServersPingAsync, Constants.Periods.DefaultServersPingInterval, Constants.Periods.DefaultServersPingInterval);
         }
 
-        public Task StartAsync() => Task.WhenAll(FixedServiceConnections.Select(c => StartCoreAsync(c)));
+        public async Task StartAsync()
+        {
+            using (new ServiceConnectionContainerScope(_serviceDiagnosticLogsContext))

```

Comment:
>ServiceConnectionContainerScope [](start = 23, length = 31)

How about those ondemand connections generated?

Reply:
call chain: `StartCoreAsync` ->`OnConnectionComplete` -> `RestartServiceConnectionCoreAsync` -> `ReplaceFixedConnections`. 
The `ServiceConnectionContainerScope` also flows there even ondemand connection is replaced the current one. So it's ok to use `ServiceConnectionContainerScope` here.